examples/fabric-admin/device_manager/DeviceSynchronization.h

```diff
@@ -65,14 +67,31 @@ class DeviceSynchronizer : public chip::app::ReadClient::Callback
     static DeviceSynchronizer & Instance();
 
 private:
+    enum class State : uint8_t
+    {
+        Idle,             ///< Default state that the object starts out in, where no work has commenced
+        Connecting,       ///< We are waiting for OnDeviceConnected or OnDeviceConnectionFailure callbacks to be called
+        AwaitingResponse, ///< We have started a subscription.
```

Comment:
would a name of `subscriptionstarted` be better given the description?

We may want to also figure out a name for receivedresponse (like subscription primed?)

Reply:
ops comment is copy and paste from somewhere else. I meant to update